ts/src/bitget.ts

```diff
@@ -2785,8 +2785,10 @@ export default class bitget extends Exchange {
         const request = {};
         let type = undefined;
         [ type, params ] = this.handleMarketTypeAndParams ('fetchTickers', market, params);
+        let subType = undefined;
+        [ subType, params ] = this.handleMarketTypeAndParams ('fetchTickers', market, params);
         let response = undefined;
-        if (type === 'spot') {
+        if (type === 'spot' && subType === undefined) {
```

Comment:
@ttodua  we should add to the docs like `params.subtype......`

Reply:
ok done here, but imo that might need a separate scope PR, where I would add those docstrings for all exchanges where they are already missing, across many methods

Reply:
@ttodua We don't need different PRs for this, if you're editing/patching a new method and you see outdated stuff (docs/safeValue/etc) you should update them right away.